src/builtins.c

```diff
@@ -524,6 +523,11 @@ JL_CALLABLE(jl_f_isdefined)
     }
     if (nargs != 2) {
         JL_NARGS(isdefined, 1, 1);
+        jl_depwarn("`isdefined(:symbol)` is deprecated, "
```

Comment:
There's a lot of it scattered everywhere. Do you really need me to write you a sed script